config/locales/en.yml

```diff
@@ -293,6 +293,7 @@ en:
       card: NEW
       section: NEW MODULE AVAILABLE
     page_count: (%{num} pages)
+    completed_all_modules: You have completed all available modules. You may use the links below to revisit any of the course content.
```

Comment:
It might be wise to move all remaining text in here now so you can work out sensible nested keys for all the sections.

Reply:
This next commit includes this update